ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let firstNameLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    private var avatarView: AvatarView
+    private var firstNameLabel: UILabel?
+    private var lastNameLabel: UILabel?
+    private var identifierLabel: UILabel?
+
+    /// Initializes the contact view with an avatar view, first name, and last name
+    ///
+    /// - Parameters:
+    ///   - avatarView: The AvatarView that will be displayed within the ContactView
+    ///   - firstName: String that will be the text of the top label
+    ///   - lastName: String that will be the text of the bottom label
+    public init(avatarView: AvatarView, firstName: String, lastName: String) {
+        self.avatarView = avatarView
+        self.firstNameLabel = UILabel()
+        self.lastNameLabel = UILabel()
+        super.init(frame: .zero)
+        self.backgroundColor = Colors.surfacePrimary
+        self.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
>self.translatesAutoresizingMaskIntoConstraints = false [](start = 8, length = 54)

why is this necessary? This is usually set before a view is added to a superview, so perhaps clients should take care of this

Reply:
Ah, gotcha. Thanks for letting me know. I've removed it.